plugins/cilium-cni/Makefile

```diff
@@ -14,7 +14,7 @@ SOURCES := $(shell find ../../api/v1/models ../../common ../../pkg/client ../../
 $(TARGET): $(SOURCES)
 	@$(ECHO_GO)
 	# Compile without cgo to allow use of cilium-cni on non-glibc platforms - see GH-5055
-	CGO_ENABLED=0 $(GO) build $(GOBUILD) -o $(TARGET) ./cilium-cni.go
+	$(QUIET)CGO_ENABLED=0 go build $(GOBUILD) -o $(TARGET) ./cilium-cni.go
```

Comment:
Why did you stop using $(GO)?

Reply:
Because `$(GO)` is just `$(QUIET)go`, which becomes `@go`, which is not a valid bash command, so it breaks `make V=0`.

I don't think I can shift the `CGO_ENABLED=0` part later in the command (I tried at end of line and it didn't work). I could alternatively add another patch which declares a new variable that maps to `go` like `$(GO_NO_MK_DIRECTIVES)` or something if you're worried that we're going to customize the `go` path.

Reply:
Resolved in latest version.